Luminary099/R63.agc

```diff
@@ -128,13 +128,13 @@ ALINEX		TC	INTPRET		# X AXIS ALIGNMENT
 # Page 340
 V89CALL1	STCALL	SCAXIS		# STORE SELECTED ALIGNMENT AXIS
 			VECPOINT	# PUTS DESIRED GIM ANG (OG,IG,MG) IN TMPAC
-		STORE	CPHI		# STOR GIMBAL ANGLES FOR BALLANGS CALL
+		STORE	CPHI		# STORE GIMBAL ANGLES FOR BALLANGS CALL.
 		EXIT
 		TC	BANKCALL
 		CADR	BALLANGS	# PUTS DESIRED BALL ANGLE IN FDAIX,Y,Z
 		CAF	VB06N18		# V 06 N 18
 		TC	BANKCALL	# NOUN 18 REFERS TO FDAIX,Y,Z
-		CADR	GOFLASH
+		CADR	GDFLASH
```

Comment:
This is an incorrect 'correction' GOFLASH is correct (as judged by its multiple occurrence throughout other source files). I submitted a commit that changes this back to GOFLASH